app/lib/features/users/actions/show_global_user_dialog.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:acter/features/users/widgets/user_info_drawer.dart';
+import 'package:flutter/material.dart';
+
+const Key userInfoDrawer = Key('users-widgets-user-info-drawer');
```

Comment:
mostly copy of the member-info-drawer section but adapted to our profile (which might even be missing) and doesn't have any room.